apps/laboratory/tests/wallet.spec.ts

```diff
@@ -19,10 +19,17 @@ testConnectedMW.afterEach(async ({ modalPage, modalValidator, walletValidator, b
 testConnectedMW(
   'it should sign',
   async ({ modalPage, walletPage, modalValidator, walletValidator }) => {
-    await modalPage.sign()
-    await walletValidator.expectReceivedSign({ chainName: DEFAULT_CHAIN_NAME })
-    await walletPage.handleRequest({ accept: true })
-    await modalValidator.expectAcceptedSign()
+    if (modalPage.library === 'solana') {
+      await modalPage.sign()
+      await walletValidator.expectReceivedSign({ chainName: 'Solana' })
+      await walletPage.handleRequest({ accept: true })
+      await modalValidator.expectAcceptedSign()
+    } else {
+      await modalPage.sign()
+      await walletValidator.expectReceivedSign({ chainName: DEFAULT_CHAIN_NAME })
+      await walletPage.handleRequest({ accept: true })
+      await modalValidator.expectAcceptedSign()
+    }
```

Comment:
I think this can be reduced to 
```
      const chainName = modalPage.library === 'solana' ? 'Solana' : DEFAULTCHAIN_NAME
      await modalPage.sign()
      await walletValidator.expectReceivedSign({ chainName })
      await walletPage.handleRequest({ accept: true })
      await modalValidator.expectAcceptedSign()
      ```